content/docs/user-guide/project-structure/configuration.md

```diff
@@ -0,0 +1,379 @@
+and their loading priority are detailed below:
+
+<!-- Avoids new lines in the Flag columns (below). -->
+<style>
+  #markdown-root td:first-child code {
+    white-space: nowrap;
+  }
+</style>
+
+| Priority | Type              | macOS location                                  | Linux location (typical\*) | Windows location                                          |
+| -------- | ----------------- | ----------------------------------------------- | -------------------------- | --------------------------------------------------------- |
+| 1        | Local             | `.dvc/config.local`                             | _N/A_                      | _N/A_                                                     |
+| 2        | Project (default) | `.dvc/config`                                   | _N/A_                      | _N/A_                                                     |
+| 3        | Global            | `$HOME/Library/Application\ Support/dvc/config` | `$HOME/.config/dvc/config` | `%LocalAppData%\iterative\dvc\config`                     |
+| 4        | System            | `/Library/Application\ Support/dvc/config`      | `/etc/xdg/dvc/config`      | `%AllUsersProfile%\Application Data\iterative\dvc\config` |
+
+<admon type="info">
+
+\* For Linux, the global file may be found in `$XDG_CONFIG_HOME`, and the system
+file in `$XDG_CONFIG_DIRS[0]`, if those env vars are defined.
+
+</admon>
+
+<admon type="tip">
+
+See also `dvc config` flags `--local`, `--global`, and `--system`.
+
+</admon>
+
+## Configuration sections
+
+The following config sections are written by this command to the appropriate
+config file (`.dvc/config` by default), supporting different config options
+within:
+
+- [`core`](#core) - main section with the general config options
+- [`remote`](#remote) - sections in the config file that describe [remote
+  storage]
+- [`cache`](#cache) - options that affect the project's <abbr>cache</abbr>
+- [`exp`](#exp) - options to change the default repo paths assumed by
+  `dvc exp init`
+- [`hydra`](#hydra) - options around [Hydra Composition] for experiment
+  configuration.
+- [`parsing`](#parsing) - options around the parsing of [dictionary unpacking].
+- [`plots`](#plots) - options for configuring `dvc plots`.
+- [`state`](#state) - see [Internal directories and files][internals] to learn
+  more about the state database.
+- [`index`](#index) - see [Internal directories and files][internals] to learn
+  more about remote index files.
+
+[remote storage]: /doc/user-guide/data-management/remote-storage
+[hydra composition]: /doc/user-guide/experiment-management/hydra-composition
+[dictionary unpacking]:
+  /doc/user-guide/project-structure/dvcyaml-files#dictionary-unpacking
+[internals]: /doc/user-guide/project-structure/internal-files
+
+### core
+
+- [`core.remote`](#remote) - name of the default remote storage
+
+- `core.interactive` - whether to always ask for confirmation before reproducing
```

Comment:
I just wonder whether these should be hidden in `<details>` sections, so the page doesn't seem that long.

Reply:
Have you tried L2 headers instead of L3 so they all show in the sidebar?

Reply:
I guess that would also be better than now, yes. I'd still also consider using hidden sections... Maybe both?

Reply:
U: I tried combining both and it does work (when you click in the title on the right sidebar it scrolls and opens the section). Up to you @dberenbaum

Reply:
p.s. updated to use both for now, PTAL.